docs/api/ethereum.md

```diff
@@ -1086,6 +1352,10 @@ Please note, when using [Infura](https://infura.io/), only queries are supported
 
 A class instance `variable` holding a `string` which represents the URL to the compiled ERC-721 related smart contract definition file. This file is used when deploying new asset ledgers to the network.
 
+### deployGatewayId
+
+A class instance `variable` holding a `string` which represents an Ethereum address of the [deploy gateway](/#public-addresses).
```

Comment:
A class instance `variable` holding a `string` that represents an Ethereum address of the [deploy gateway](/#public-addresses).